src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs

```diff
@@ -39,7 +39,7 @@ private void Context_PropertyChanged(object? sender, PropertyChangedEventArgs e)
 		{
 			if (e.PropertyName is nameof(IContentPageContext.HasSelection))
 			{
-				if (context.ShellPage is not null)
+				if (context.ShellPage is not null && context.ShellPage.SlimContentPage is not null)
```

Comment:
What's the difference between ShellPage and SlimContentPage?

Reply:
`ShellPage` is `Modern` or `Column`, `SlimeContentPage` refers to the layout in use
I think they are not synced and so one is updated while the other is not and remains null.
I had found this bug while working on #11673 and I had fixed it there too.

Reply:
Thanks! We really need to move away from this concept but that's not your fault.